ledger/src/blockstore_options.rs

```diff
@@ -135,13 +135,24 @@ impl Default for ShredStorageType {
     }
 }
 
+pub const BLOCKSTORE_DIRECTORY_ROCKS_LEVEL: &str = "rocksdb";
+pub const BLOCKSTORE_DIRECTORY_ROCKS_FIFO: &str = "rocksdb_fifo";
```

Comment:
What do you think about not making these `pub`, and instead replacing the remaining instances of these constants  (looks like only `Blockstore::destroy()`) with calls like `ShredStorageType::RocksLevel.blockstore_directory()` (similar to what you did in the unit tests).